.circleci/config.yml

```diff
@@ -314,7 +314,7 @@ workflows:
           matrix:
             parameters:
               python_version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-              xcode_version: ["15.0.0", "15.2.0"]
+              xcode_version: ["15.0.0", "15.2.0", "16.0.0"]
```

Comment:
I'm not actually sure if this will work for the release. It needs to be tested. It might make the same wheel (14.0) as using xcode 15.2.0

Reply:
Actually it puts 15_0 in the wheel so should be good:
`mlx-0.17.2.dev20240831+161b2e8f-cp312-cp312-macosx_15_0_arm64.whl`